src/components/ConversionFormOverlay/style.css

```diff
@@ -0,0 +1,56 @@
+.container{
+    height: 100vh;
+    width: 100%;
+    background-color: rgba(231, 228, 242, 1);
+    padding-left: 30px;  
+    padding-top: 30px;
+}
+
+.container .loading-icon, .container.loading-text {
+    text-align: center;
+}
+
+.upload {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+
+.upload-container {
+    display: flex;
+    flex-direction: row;
+}
+
+.selector-box {
+    width: 200px;
+    margin-right: 20px;
+}
+
+.container button {
+    margin-right: 20px;
+    min-width: 80px;
+}
+
+.container span svg path {
+    fill: black
+}
```

Comment:
add a note that this is for the dropdown arrow since this is a little odd otherwise. It should also be our theme dark text color, not just hardcoded black